samples/azure/native-integration-asb/ASB_7/MessageGenerator/packages.config

```diff
@@ -1,8 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="Newtonsoft.Json" version="9.0.1" targetFramework="net452" />
-  <package id="NServiceBus" version="6.0.3" targetFramework="net461" />
```

Comment:
Did you pick the wrong side of comparison. Seems to indicate a version down grade?